### ISSUE TITLE
BUG: Go's queues/stacks fails

### ISSUE THREAD

**Issue by geekygirlsarah** [codethesaurus/codethesaur.us#689]
Error report:

```
Internal Server Error: /reference/

AttributeError at /reference/
'MetaInfo' object has no attribute 'language_friendly_name'

Request Method: GET
Request URL: https://codethesaur.us/reference/?concept=queues_stacks&lang=go%3B1
Django Version: 4.2.3
Python Executable: /app/.heroku/python/bin/python
Python Version: 3.11.4
Python Path: ['/app/.heroku/python/bin', '/app', '/app/.heroku/python/lib/python311.zip', '/app/.heroku/python/lib/python3.11', '/app/.heroku/python/lib/python3.11/lib-dynload', '/app/.heroku/python/lib/python3.11/site-packages']
Server time: Tue, 24 Oct 2023 15:22:05 +0000
Installed Applications:
['web.apps.WebConfig',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'markdownify.apps.MarkdownifyConfig']
Installed Middleware:
('django.middleware.security.SecurityMiddleware',
 'whitenoise.middleware.WhiteNoiseMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.common.BrokenLinkEmailsMiddleware')


Traceback (most recent call last):
  File "/app/web/models.py", line 278, in load_languages
    language.load_concepts(meta_structure.key, version)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/web/models.py", line 91, in load_concepts
    with open(file_path, 'r', encoding='UTF-8') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

During handling of the above exception ([Errno 2] No such file or directory: 'web/thesauruses/go/1/queues_stacks.json'), another exception occurred:
  File "/app/.heroku/python/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/app/.heroku/python/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/web/views.py", line 153, in concepts
    languages = meta_info.load_languages(language_strings, meta_structure)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/web/models.py", line 284, in load_languages
    self.language_friendly_name(language_key),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^

Exception Type: AttributeError at /reference/
Exception Value: 'MetaInfo' object has no attribute 'language_friendly_name'
Raised during: web.views.concepts
Request information:
USER: AnonymousUser
```

**Comment by JureBl** [comment]
Can you assign me?

**Comment by geekygirlsarah** [comment]
Sure! Have fun!

I don't know anything besides that popped in as a server error after someone went to the URL above. I have no context, so it's a bit more vague. Good luck and let me know if you have questions!

**Comment by JureBl** [comment]
I started the project on my local machine and it worked fine.  I couldn't replicate the problem

**Comment by JureBl** [comment]
Oh no same problem on my local machine

**Comment by geekygirlsarah** [comment]
Closing this as @JureBl fixed it in #690 but changed the line "fixes #690" so it didn't auto-close.
